server/server/src/utils.rs

```diff
@@ -18,3 +21,37 @@ pub fn get_build_info() -> Result<GetBuildInfoResponse, ServerError<GetBuildInfo
         git_commit_hash: build_info::COMMIT_HASH,
     })
 }
+
+pub async fn get_android_client(config: &Config) -> AndroidPublisher {
+    let auth = match &config.google.service_account_key {
+        Some(key) => {
+            let service_account_key: oauth2::ServiceAccountKey =
+                oauth2::parse_service_account_key(key).unwrap();
+
+            oauth2::ServiceAccountAuthenticator::builder(service_account_key)
+                .build()
+                .await
+                .unwrap()
+        }
+        None => {
+            // creating dummy AndroidPublisher since no service account was provided
+            oauth2::InstalledFlowAuthenticator::builder(
+                oauth2::ApplicationSecret::default(),
+                oauth2::InstalledFlowReturnMethod::HTTPRedirect,
+            )
+            .build()
+            .await
+            .unwrap()
+        }
+    };
+
+    let client = hyper::Client::builder().build(
+        hyper_rustls::HttpsConnectorBuilder::with_native_roots(Default::default())
+            .https_or_http()
+            .enable_http1()
+            .enable_http2()
+            .build(),
+    );
+
+    AndroidPublisher::new(client, auth)
+}
```

Comment:
This doesn't seem like the right place for this. The connection logic for the clients is usually alongside those clients.